pkg/nimble/glue/controller_init.c

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
In some other packages (openthread/semtech-loramac/...), RIOT specific/glue code is located in a `contrib` subdirectory, maybe use the same here ?